### PR TITLE
Add popup dialogs for right mouse click in resolution dialog

### PR DESCRIPTION
**Pull request by ihhub**
To be consistent with the rest of the code

### CODE REVIEW

File: src/fheroes2/dialog/dialog_resolution.cpp [hunk 1]
```diff
@@ -182,7 +184,18 @@ namespace Dialog
...
             }
-            else if ( resList.isSelected() ) {
+            else if ( le.MousePressRight( buttonCancel.area() ) ) {
+                fheroes2::Text header( _( "Cancel" ), { fheroes2::FontSize::NORMAL, fheroes2::FontColor::YELLOW } );
+                fheroes2::Text body( _( "Exit this menu without doing anything." ), { fheroes2::FontSize::NORMAL, fheroes2::FontColor::WHITE } );
+                fheroes2::showMessage( header, body, 0 );
+            }
+            else if ( le.MousePressRight( buttonOk.area() ) ) {
+                fheroes2::Text header( _( "Okay" ), { fheroes2::FontSize::NORMAL, fheroes2::FontColor::YELLOW } );
+                fheroes2::Text body( _( "Click to apply selected resolution." ), { fheroes2::FontSize::NORMAL, fheroes2::FontColor::WHITE } );
```
Comment (oleg-derevenetz) [inline src/fheroes2/dialog/dialog_resolution.cpp:194]: I believe there should be "the": "Click to apply the selected resolution" :)

**Review by oleg-derevenetz — CHANGES_REQUESTED** [review]
Hi @ihhub I only found one typo, please take a look when you have a free time.

File: src/fheroes2/dialog/dialog_resolution.cpp (hunk 1 above)
Comment (ihhub) [reply src/fheroes2/dialog/dialog_resolution.cpp:194]: You're right.

Comment (ihhub) [reply src/fheroes2/dialog/dialog_resolution.cpp]: Fixed.

**Review by oleg-derevenetz — APPROVED** [review]

**Pull request merged**